cdap-proto/src/main/java/co/cask/cdap/proto/WorkflowStatistics.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class that is used to return the statistics of a workflow
+ */
+public class WorkflowStatistics {
+  private final long startTime;
+  private final long endTime;
+  private final int runs;
+  private final double avgRunTime;
+  private final List<PercentileInformation> percentileInformationList;
+  private final Map<String, Map<String, Double>> actionToStatistic;
+
+  public WorkflowStatistics(long startTime, long endTime, int runs, double avgRunTime,
+                            List<PercentileInformation> percentileInformationList,
+                            Map<String, Map<String, Double>> actionToStatistic) {
+    this.startTime = startTime;
+    this.endTime = endTime;
+    this.runs = runs;
+    this.avgRunTime = avgRunTime;
+    this.percentileInformationList = percentileInformationList;
+    this.actionToStatistic = actionToStatistic;
+  }
+
+  public List<PercentileInformation> getPercentileInformationList() {
+    return percentileInformationList;
+  }
+
+  public int getRuns() {
+    return runs;
+  }
+
+  public double getAvgRunTime() {
+    return avgRunTime;
+  }
+
+  public long getStartTime() {
+    return startTime;
+  }
+
+  public long getEndTime() {
+    return endTime;
+  }
+
+  public Map<String, Map<String, Double>> getActionToStatistic() {
```

Comment:
since ``actionToStatistic`` is not very clear, will be useful to include the comment above with a sample response.